OpenRA.Game/Network/ChatLine.cs

```diff
@@ -0,0 +1,55 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2021 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using OpenRA.Primitives;
+
+namespace OpenRA.Network
+{
+	public enum ChatPool { System, Chat, Mission, Feedback, Transcriptions }
+	public class ChatLine : IEquatable<ChatLine>
+	{
+		public readonly Color NameColor;
+		public readonly string Name;
+		public readonly string Text;
+		public readonly Color TextColor;
+		public readonly ChatPool Pool;
+
+		public ChatLine(string name, Color nameColor, string text, Color textColor, ChatPool chatPool)
+		{
+			NameColor = nameColor;
+			Name = name;
+			Text = text;
+			TextColor = textColor;
+			Pool = chatPool;
+		}
+
+		public bool IncrementOnDuplicate()
```

Comment:
```suggestion
		public bool CanIncrementOnDuplicate()
```
`IncrementOnDuplicate` sounds like it will do the increment.

Reply:
Actually this is from #18905.